javascript/code-challenges/challenges-03.test.js

```diff
@@ -38,9 +38,8 @@ Write a function named containsAnd that, given an array of strings as input, use
 For example, containsAnd(['panda', 'ran', 'and']) returns ['panda', 'and'].
 ------------------------------------------------------------------------------------------------ */
 
-const containsAnd = (arr) => {
-  return arr.filter(and => and.includes('and')); //found similar solution on stack overflow while searching .filter
-};
+const containsAnd = (arr) => arr.filter(str => str.includes('and')); //found similar solution on stack overflow while searching .filter
```

Comment:
Also way to try and make it a one liner. Almost every code challenge from today can be written short like this.